frontend/src/metabase/dashboard/components/DashCard/DashCardActionsPanel/ChartSettingsButton/ChartSettingsButton.tsx

```diff
@@ -50,7 +50,7 @@ export function ChartSettingsButton({
         <Modal.Overlay />
         <Modal.Content mih="85%">
           <Modal.Body>
-            <ChartSettingsWithState
+            <DashboardChartSettings
```

Comment:
This is a great change. I checked that this component is only used here, so it makes sense to name it more specific than generic.